SU2_CFD/src/output_adjoint_heat.cpp

```diff
@@ -0,0 +1,206 @@
+/*!
+ * \file output_adjoint_heat.cpp
+ * \brief Main subroutines for flow discrete adjoint output
+ * \author R. Sanchez
+ * \version 6.0.1 "Falcon"
+ *
+ * The current SU2 release has been coordinated by the
+ * SU2 International Developers Society <www.su2devsociety.org>
+ * with selected contributions from the open-source community.
+ *
+ * The main research teams contributing to the current release are:
+ *  - Prof. Juan J. Alonso's group at Stanford University.
+ *  - Prof. Piero Colonna's group at Delft University of Technology.
+ *  - Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+ *  - Prof. Alberto Guardone's group at Polytechnic University of Milan.
+ *  - Prof. Rafael Palacios' group at Imperial College London.
+ *  - Prof. Vincent Terrapon's group at the University of Liege.
+ *  - Prof. Edwin van der Weide's group at the University of Twente.
+ *  - Lab. of New Concepts in Aeronautics at Tech. Institute of Aeronautics.
+ *
+ * Copyright 2012-2018, Francisco D. Palacios, Thomas D. Economon,
+ *                      Tim Albring, and the SU2 contributors.
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../include/output/output_heat_discadj.hpp"
+
+CDiscAdjHeatOutput::CDiscAdjHeatOutput(CConfig *config, CGeometry *geometry, unsigned short val_iZone) : COutput(config) {
+  
+  nDim = geometry->GetnDim();
+ 
+  turb_model = config->GetKind_Turb_Model();
+  
+  heat = config->GetEnergy_Equation();
+  
+  weakly_coupled_heat = config->GetWeakly_Coupled_Heat();
+  
+  /*--- Set the default history fields if nothing is set in the config file ---*/
+  
+  if (nRequestedHistoryFields == 0){
+    RequestedHistoryFields.push_back("ITER");
```

Comment:
please use emplace_back here to be consistent with the rest of the output

Reply:
`SU2_CFD/src/output_adjoint_heat.cpp` for some reason was still traced by git, it's old and removed now.